modules/tensor_mechanics/doc/content/modules/tensor_mechanics/FractureIntegrals.md

```diff
@@ -70,9 +70,11 @@ To use the $J$-integral capability in MOOSE, a user specifies a set of nodes alo
 \begin{equation}
-\bar{I}(s) = \int_V \left[ \sigma_{ij} u_{j,1}^{(aux)} + \sigma_{ij}^{(aux)} u_{j,1} - \sigma_{jk} \epsilon_{jk}^{(aux)} \delta_{1i} \right] q_{,i} \, \mathrm{d}V + \int_V \left[ \epsilon^*_{ij,1} \sigma_{ij}^{(aux)} - b_i u_{i,1}^{(aux)} \right] q \, \mathrm{d}V
+\bar{I}(s) = \int_V \left[ \sigma_{ij} u_{j,1}^{(aux)} + \sigma_{ij}^{(aux)} u_{j,1} - \sigma_{jk} \epsilon_{jk}^{(aux)} \delta_{1i} \right] q_{,i} \, \mathrm{d}V + \int_V \left[ \epsilon^*_{ij,1} \sigma_{ij}^{(aux)} - b_i u_{i,1}^{(aux)} \right] q \ + \mathrm{d}V +\int_V \left[ \sigma_{ij,j}^{(aux)}u_{i,1}^{(aux)} - C_{ijkl,1}\epsilon_{kl} \epsilon_{ij}^{(aux)} \right] q \, \mathrm{d}V,
 \end{equation}
-where $\sigma$ is the stress, $u$ is the displacement, and $q$ is identical to the $q$-functions used for $J$-integrals. The last two terms in this equation account for the effects of the gradient of the eigenstrain, $\epsilon^*$, and for the effects of the body force, $b$. If the eigenstrain is a thermal strain, its gradient is computed using the gradient of the temperature field and the temperature derivative of the eigenstrain. In that case, the user supplies the temperature and eigenstrain name through the `temperature` and `eigenstrain_name` parameters, respectively. If the eigenstrain is from another source, its gradient must be computed by the eigenstrain material model in the form of a RankThreeTensor object, whose name is specified for the fracture integrals using the `eigenstrain_gradient` parameter.
+where $\sigma$ is the stress, $u$ is the displacement, and $q$ is identical to the $q$-functions used for $J$-integrals. The second integral in this equation accounts for the effects of the gradient of the eigenstrain, $\epsilon^*$, and for the effects of the body force, $b$. If the eigenstrain is a thermal strain, its gradient is computed using the gradient of the temperature field and the temperature derivative of the eigenstrain. In that case, the user supplies the temperature and eigenstrain name through the `temperature` and `eigenstrain_name` parameters, respectively. If the eigenstrain is from another source, its gradient must be computed by the eigenstrain material model in the form of a RankThreeTensor object, whose name is specified for the fracture integrals using the `eigenstrain_gradient` parameter. The last integral accounts for spatially-varying elastic properties in the material using a "non-equilibrium" formulation. The user can define an auxiliary variable over a spatial area over which the elasticity modulus of the material transitions, i.e. $E(X_{1}) = E_{0} \exp{\beta X_{1}}$, in which $\beta$ is a parameter that governs the transition of the material parameter.
```

Comment:
Suggested re-wording for lines 75 and 77:

```
where $\sigma$ is the stress, $u$ is the displacement, and $q$ is identical to the $q$-functions used for $J$-integrals. The second integral in this equation accounts for the effects of the gradient of the eigenstrain, $\epsilon^*$, and for the effects of the body force, $b$. If the eigenstrain is a thermal strain, its gradient is computed using the gradient of the temperature field and the temperature derivative of the eigenstrain. In that case, the user supplies the temperature and eigenstrain name through the `temperature` and `eigenstrain_name` parameters, respectively. If the eigenstrain is from another source, its gradient must be computed by the eigenstrain material model in the form of a RankThreeTensor object, whose name is specified for the fracture integrals using the `eigenstrain_gradient` parameter. The last integral accounts for spatially-varying elastic properties in the material (such as for functionally graded materials) using a "non-equilibrium" formulation. 

The treatment for functionally graded materials is currently limited to cases for which the gradient of the Young's modulus is in the direction of crack extension. For example, this could be used to represent a crack that is perpendicular to an interface between two materials, where the transition between the materials is assumed to be approximated by a smooth function such as $E(X_{1}) = E_{0} \exp{\beta X_{1}}$, where $X_{1}$ is the position in the direction of crack extension, $E_{0}$ is the stiffness of the material behind the crack tip, and $\beta$ is a parameter that governs the rate of the material property change. To specify the spatially varying properties, the user must define the `functionally_graded_youngs_modulus` and `functionally_graded_youngs_modulus_crack_dir_gradient` parameters, which reference scalar material properties that define the spatially varying Young's modulus and its gradient in the crack direction, respectively.  These material properties can be defined arbitrarily by the user, but it is up to the user to ensure that the gradient is indeed in the crack direction. Note that the user must still also specify `youngs_modulus` to define that moduls at the crack tip. See the "non-equilibrium" formulation in Ref.~[!cite](Jeong2004functionallygraded) for details.